src/plugins/vis_type_xy/public/components/detailed_tooltip.mock.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const getAspects = () => {
+  return {
+    x: {
+      accessor: 'col-0-3',
+      column: 0,
+      title: 'timestamp per 3 hours',
+      format: {
+        id: 'date',
+        params: {
+          pattern: 'YYYY-MM-DD HH:mm',
+        },
+      },
+      aggType: 'date_histogram',
+      aggId: '3',
+      params: {
+        date: true,
+        intervalESUnit: 'h',
+        intervalESValue: 3,
+        interval: 10800000,
+        format: 'YYYY-MM-DD HH:mm',
+      },
+    },
+    y: [
+      {
+        accessor: 'col-1-1',
+        column: 1,
+        title: 'Count',
+        format: {
+          id: 'number',
+        },
+        aggType: 'count',
+        aggId: '1',
+        params: {},
+      },
+    ],
+  };
+};
+
+export const getAspectsWithSplitColumn = () => {
+  return {
+    x: {
+      accessor: 'col-0-3',
+      column: 0,
+      title: 'timestamp per 3 hours',
+      format: {
+        id: 'date',
+        params: {
+          pattern: 'YYYY-MM-DD HH:mm',
+        },
+      },
+      aggType: 'date_histogram',
+      aggId: '3',
+      params: {
+        date: true,
+        intervalESUnit: 'h',
+        intervalESValue: 3,
+        interval: 10800000,
+        format: 'YYYY-MM-DD HH:mm',
+      },
+    },
+    y: [
+      {
+        accessor: 'col-2-1',
+        column: 2,
+        title: 'Count',
+        format: {
+          id: 'number',
+        },
+        aggType: 'count',
+        aggId: '1',
+        params: {},
+      },
+    ],
+    splitColumn: {
+      accessor: 'col-1-4',
+      column: 1,
+      title: 'Cancelled: Descending',
+      format: {
+        id: 'terms',
+        params: {
+          id: 'boolean',
+          otherBucketLabel: 'Other',
+          missingBucketLabel: 'Missing',
+        },
+      },
+      aggType: 'terms',
+      aggId: '4',
+      params: {},
+    },
+  };
+};
+
+export const getAspectsWithSplitRow = () => {
```

Comment:
it's it ok that `getAspectsWithSplitColumn` and `getAspectsWithSplitRows` contain absolutely the same mocked data?  I think we can modify it at least to cover more scenarios. Also that will help to avoid of duplication code warning in `IDE`.